sycl/source/detail/scheduler/commands.hpp

```diff
@@ -193,7 +220,10 @@ class Command {
   /// Counts the number of memory objects this command is a leaf for.
   unsigned MLeafCounter = 0;
 
-  const char *MBlockReason = "Unknown";
+  enum class BlockReason : int { HostAccessor = 0, HostTask };
```

Comment:
At some point we will need to reduce the size of all these enum,
A good opportunity to start with
```c++
enum class BlockReason : char { HostAccessor = 0, HostTask };
```